<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="Adjusting genetic correlation estimates to account for the genetics of a common confounder can help better understand the role of this confounder in explaining the genetic similarity between conditions. This approach provides a way to estimate and compare unadjusted and partial genetic correlation estimates. ">
<title>R package to estimate partial genetic correlations • partialLDSC</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="apple-touch-icon-60x60.png">
<script src="deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="pkgdown.js"></script><meta property="og:title" content="R package to estimate partial genetic correlations">
<meta property="og:description" content="Adjusting genetic correlation estimates to account for the genetics of a common confounder can help better understand the role of this confounder in explaining the genetic similarity between conditions. This approach provides a way to estimate and compare unadjusted and partial genetic correlation estimates. ">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="index.html">partialLDSC</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.1</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="reference/index.html">Reference</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="news/index.html">Changelog</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav"></ul>
</div>

    
  </div>
</nav><div class="container template-home">
<div class="row">
  <main id="main" class="col-md-9"><div class="section level1">
<div class="page-header"><h1 id="partialldsc-">partialLDSC <a href="https://github.com/GEMINI-multimorbidity/partialLDSC" class="external-link"><img src="reference/figures/hex.png" align="right" width="150"></a>
<a class="anchor" aria-label="anchor" href="#partialldsc-"></a>
</h1></div>
<!-- badges: start -->

<p>For details on the method see our <a href="https://doi.org/10.1101/2024.07.10.24309772" class="external-link">preprint</a>. If you use the package <code>partialLDSC</code> please cite:</p>
<blockquote>
<p>Mounier <em>et al.</em> (2024) Genetics identifies obesity as a shared risk factor for co-occurring multiple long-term conditions. medRxiv <a href="https://doi.org/10.1101/2024.07.10.24309772" class="external-link">https://doi.org/10.1101/2024.07.10.24309772</a></p>
</blockquote>
<div class="section level2">
<h2 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<p><code>partialLDSC</code> is an R-package to estimate partial genetic correlations from GWAS summary statistics, and compare them to their unadjusted counterparts, to quantify the contribution of a given confounder in explaining genetic similarity between conditions. The partial genetic correlations between two conditions correspond to their genetic correlation, holding the genetic effects of a potential confounder constant. Differences between unadjusted and partial estimates are not necessarily due to a causal effect of the potential confounder on both conditions and further (causal inference) analyses might be needed to better describe the relationship between the conditions and the potential confounder.</p>
<p>It relies on cross-trait LD-score regression (LDSC), as first described by <a href="https://pubmed.ncbi.nlm.nih.gov/26414676/" class="external-link">Bulik-Sullivan, B. et al. - “An atlas of genetic correlations across human diseases and traits.”</a>.</p>
<p>Our implementation of LDSC is based on the one from <a href="https://github.com/GenomicSEM/GenomicSEM/" class="external-link"><code>GenomicSEM</code></a>. Moreover, the pre-processing of the GWAS summary statistics prior to analysis should be done using the <code>munge</code> function they provide.</p>
<p>There are two main functions available:</p>
<ul>
<li><p><strong><code><a href="reference/partial_ldsc.html">partial_ldsc()</a></code></strong><br>
main function to estimate unadjusted and partial genetic correlations (as well as heritabilities, on the observed scale only), and compare them to each other to assess if adjusting for the potential confounder’s genetic significantly affect the pairwise genetic correlation estimates.</p></li>
<li><p><strong><code><a href="reference/forest_plot.html">forest_plot()</a></code></strong><br>
main function to visualise the results.</p></li>
</ul>
<p>More details about their usage can be found in the <a href="doc/partialLDSC-manual.pdf">manual</a>.</p>
</div>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p>You can install the current version of <code>partialLDSC</code> with:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Directly install the package from github</span></span>
<span><span class="co"># install.packages("remotes")</span></span>
<span><span class="fu">remotes</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"GEMINI-multimorbidity/partialLDSC"</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">partialLDSC</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="usage">Usage<a class="anchor" aria-label="anchor" href="#usage"></a>
</h2>
<p>To run the analysis with <code>partialLDSC</code> different inputs are needed:</p>
<div class="section level4">
<h4 id="id_1-the-munged-gwas-summary-statistics-conditions--confounder">1. The munged GWAS summary statistics (<code>conditions</code> &amp; <code>confounder</code>):<a class="anchor" aria-label="anchor" href="#id_1-the-munged-gwas-summary-statistics-conditions--confounder"></a>
</h4>
<p>More information about how to munge the summary statistics can be found in the <a href="https://github.com/GenomicSEM/GenomicSEM/" class="external-link"><code>GenomicSEM</code></a> R-package documentaion, <a href="https://github.com/GenomicSEM/GenomicSEM/wiki/3.-Models-without-Individual-SNP-effects#step-1-munge-the-summary-statistics" class="external-link">here</a>.</p>
</div>
<div class="section level4">
<h4 id="id_2-the-input-files-for-ldsc-ld">2. The input files for LDSC (<code>ld</code>):<a class="anchor" aria-label="anchor" href="#id_2-the-input-files-for-ldsc-ld"></a>
</h4>
<p>LD scores are needed, these are the same as the ones needed by the <a href="https://github.com/GenomicSEM/GenomicSEM/" class="external-link"><code>GenomicSEM</code></a> R-package and can be directly downloaded from the link they provide.</p>
<blockquote>
<p>Expects LD scores formated as required by the original LD score regression software. Weights for the european population can be obtained by downloading the eur_w_ld_chr folder in the link below (Note that these are the same weights provided by the original developers of LDSC): <a href="https://utexas.box.com/s/vkd36n197m8klbaio3yzoxsee6sxo11v" class="external-link uri">https://utexas.box.com/s/vkd36n197m8klbaio3yzoxsee6sxo11v</a></p>
</blockquote>
</div>
<div class="section level3">
<h3 id="analysis">Analysis<a class="anchor" aria-label="anchor" href="#analysis"></a>
</h3>
<p>Before running the examples, please make sure to have downloaded the ld-scores files. You may also need to modify the <code>ld</code> parameters to indicate the correct path. Note that when running the analysis with your own GWAS summary statistics, you will first need to properly munge them.</p>
<ul>
<li><strong>Example A</strong></li>
</ul>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Using GEMINI GWAS summary statistics for three conditions </span></span>
<span><span class="co"># (osteoarthitis: OA, type 2 diabetes: T2D, benign hyperplasia of prostate: BPH)</span></span>
<span><span class="co"># + GIANT GWAS summary statistics for the confounder (BMI)</span></span>
<span><span class="co"># (1,150,000 SNPs - stored in gzipped files)</span></span>
<span><span class="va">OA_file</span>  <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"Data/"</span>, <span class="st">"OA_GEMINI.sumstats.gz"</span>, package<span class="op">=</span><span class="st">"partialLDSC"</span><span class="op">)</span></span>
<span><span class="va">T2D_file</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"Data/"</span>, <span class="st">"diabetes_type_2_GEMINI.sumstats.gz"</span>, package<span class="op">=</span><span class="st">"partialLDSC"</span><span class="op">)</span></span>
<span><span class="va">BPH_file</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"Data/"</span>, <span class="st">"BPH_GEMINI.sumstats.gz"</span>, package<span class="op">=</span><span class="st">"partialLDSC"</span><span class="op">)</span></span>
<span><span class="va">BMI_file</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"Data/"</span>, <span class="st">"BMI_Yengo_2018.txt.sumstats.gz"</span>, package<span class="op">=</span><span class="st">"partialLDSC"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># launch analysis (using default number of blocks)</span></span>
<span><span class="va">A</span> <span class="op">=</span> <span class="fu"><a href="reference/partial_ldsc.html">partial_ldsc</a></span><span class="op">(</span>conditions <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">OA_file</span>, <span class="va">T2D_file</span>, <span class="va">BPH_file</span><span class="op">)</span>,</span>
<span>                 confounder <span class="op">=</span> <span class="va">BMI_file</span>, </span>
<span>                 condition.names <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"OA"</span>, <span class="st">"T2D"</span>, <span class="st">"BPH"</span><span class="op">)</span>, </span>
<span>                 confounder.name <span class="op">=</span> <span class="st">"BMI"</span>,</span>
<span>                 ld <span class="op">=</span> <span class="st">"~/eur_w_ld_chr"</span>,</span>
<span>                 log.name <span class="op">=</span> <span class="st">"Example_A"</span><span class="op">)</span></span></code></pre></div>
<details><summary>
Show log
</summary><div class="sourceCode" id="cb3"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" tabindex="-1"></a><span class="do">## Multivariate ld-score regression of 4 traits (3 conditions: OA, T2D, BPH + confounder: BMI) began at: 15:13</span></span>
<span id="cb3-2"><a href="#cb3-2" tabindex="-1"></a><span class="do">## Number of blocks used to perform the block jacknife used to estimate the sampling covariance matrix (V) is 200</span></span>
<span id="cb3-3"><a href="#cb3-3" tabindex="-1"></a><span class="do">## Reading in LD scores from: C:\Users\nm572\OneDrive - University of Exeter\Documents\Exeter\Projects\Data\eur_w_ld_chr</span></span>
<span id="cb3-4"><a href="#cb3-4" tabindex="-1"></a><span class="do">## Read in summary statistics [1/4] (OA) from: C:\Users\nm572\OneDrive - University of Exeter\Documents\R\win-library\4.1\partialLDSC\Data\OA_GEMINI.sumstats.gz</span></span>
<span id="cb3-5"><a href="#cb3-5" tabindex="-1"></a><span class="do">## Out of 1168143 SNPs, 1155445 remain after merging with LD-score files</span></span>
<span id="cb3-6"><a href="#cb3-6" tabindex="-1"></a><span class="do">## Removing 0 SNPs with Chi^2 &gt; 766.622; 1155445 remain</span></span>
<span id="cb3-7"><a href="#cb3-7" tabindex="-1"></a><span class="do">## Read in summary statistics [2/4] (T2D) from: C:\Users\nm572\OneDrive - University of Exeter\Documents\R\win-library\4.1\partialLDSC\Data\diabetes_type_2_GEMINI.sumstats.gz</span></span>
<span id="cb3-8"><a href="#cb3-8" tabindex="-1"></a><span class="do">## Out of 1161441 SNPs, 1153850 remain after merging with LD-score files</span></span>
<span id="cb3-9"><a href="#cb3-9" tabindex="-1"></a><span class="do">## Removing 15 SNPs with Chi^2 &gt; 446.142; 1153835 remain</span></span>
<span id="cb3-10"><a href="#cb3-10" tabindex="-1"></a><span class="do">## Read in summary statistics [3/4] (BPH) from: C:\Users\nm572\OneDrive - University of Exeter\Documents\R\win-library\4.1\partialLDSC\Data\BPH_GEMINI.sumstats.gz</span></span>
<span id="cb3-11"><a href="#cb3-11" tabindex="-1"></a><span class="do">## Out of 1166954 SNPs, 1160509 remain after merging with LD-score files</span></span>
<span id="cb3-12"><a href="#cb3-12" tabindex="-1"></a><span class="do">## Removing 1 SNPs with Chi^2 &gt; 203.231; 1160508 remain</span></span>
<span id="cb3-13"><a href="#cb3-13" tabindex="-1"></a><span class="do">## Read in summary statistics [4/4] (BMI) from: C:\Users\nm572\OneDrive - University of Exeter\Documents\R\win-library\4.1\partialLDSC\Data\BMI_Yengo_2018.txt.sumstats.gz</span></span>
<span id="cb3-14"><a href="#cb3-14" tabindex="-1"></a><span class="do">## Out of 1018130 SNPs, 1014995 remain after merging with LD-score files</span></span>
<span id="cb3-15"><a href="#cb3-15" tabindex="-1"></a><span class="do">## Removing 6 SNPs with Chi^2 &gt; 795.64; 1014989 remain</span></span>
<span id="cb3-16"><a href="#cb3-16" tabindex="-1"></a><span class="do">##           </span></span>
<span id="cb3-17"><a href="#cb3-17" tabindex="-1"></a><span class="do">## Estimating heritability [1/10] for: OA</span></span>
<span id="cb3-18"><a href="#cb3-18" tabindex="-1"></a><span class="do">## Heritability Results for trait: OA</span></span>
<span id="cb3-19"><a href="#cb3-19" tabindex="-1"></a><span class="do">## Mean Chi^2 across remaining SNPs: 1.4953</span></span>
<span id="cb3-20"><a href="#cb3-20" tabindex="-1"></a><span class="do">## Lambda GC: 1.22</span></span>
<span id="cb3-21"><a href="#cb3-21" tabindex="-1"></a><span class="do">## Intercept: 0.8991 (0.0113)</span></span>
<span id="cb3-22"><a href="#cb3-22" tabindex="-1"></a><span class="do">## Ratio: -0.2036 (0.0229)</span></span>
<span id="cb3-23"><a href="#cb3-23" tabindex="-1"></a><span class="do">## Total Observed Scale h2: 0.039 (0.0014)</span></span>
<span id="cb3-24"><a href="#cb3-24" tabindex="-1"></a><span class="do">## h2 Z: 28</span></span>
<span id="cb3-25"><a href="#cb3-25" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-26"><a href="#cb3-26" tabindex="-1"></a><span class="do">## Calculating genetic covariance [2/10] for traits: OA and T2D</span></span>
<span id="cb3-27"><a href="#cb3-27" tabindex="-1"></a><span class="do">## 1145694 SNPs remain after merging OA and T2D summary statistics</span></span>
<span id="cb3-28"><a href="#cb3-28" tabindex="-1"></a><span class="do">## Results for genetic covariance between: OA and T2D</span></span>
<span id="cb3-29"><a href="#cb3-29" tabindex="-1"></a><span class="do">## Mean Z*Z: 0.2186</span></span>
<span id="cb3-30"><a href="#cb3-30" tabindex="-1"></a><span class="do">## Cross trait Intercept: 0.0299 (0.0076)</span></span>
<span id="cb3-31"><a href="#cb3-31" tabindex="-1"></a><span class="do">## Total Observed Scale Genetic Covariance (g_cov): 0.0158 (0.0016)</span></span>
<span id="cb3-32"><a href="#cb3-32" tabindex="-1"></a><span class="do">## g_cov Z: 10.2</span></span>
<span id="cb3-33"><a href="#cb3-33" tabindex="-1"></a><span class="do">## g_cov P-value: 3.0229e-24</span></span>
<span id="cb3-34"><a href="#cb3-34" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-35"><a href="#cb3-35" tabindex="-1"></a><span class="do">## Calculating genetic covariance [3/10] for traits: OA and BPH</span></span>
<span id="cb3-36"><a href="#cb3-36" tabindex="-1"></a><span class="do">## 1151912 SNPs remain after merging OA and BPH summary statistics</span></span>
<span id="cb3-37"><a href="#cb3-37" tabindex="-1"></a><span class="do">## Results for genetic covariance between: OA and BPH</span></span>
<span id="cb3-38"><a href="#cb3-38" tabindex="-1"></a><span class="do">## Mean Z*Z: 0.0934</span></span>
<span id="cb3-39"><a href="#cb3-39" tabindex="-1"></a><span class="do">## Cross trait Intercept: 0.0418 (0.0058)</span></span>
<span id="cb3-40"><a href="#cb3-40" tabindex="-1"></a><span class="do">## Total Observed Scale Genetic Covariance (g_cov): 0.0064 (0.0016)</span></span>
<span id="cb3-41"><a href="#cb3-41" tabindex="-1"></a><span class="do">## g_cov Z: 3.96</span></span>
<span id="cb3-42"><a href="#cb3-42" tabindex="-1"></a><span class="do">## g_cov P-value: 7.3987e-05</span></span>
<span id="cb3-43"><a href="#cb3-43" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-44"><a href="#cb3-44" tabindex="-1"></a><span class="do">## Calculating genetic covariance [4/10] for traits: OA and BMI</span></span>
<span id="cb3-45"><a href="#cb3-45" tabindex="-1"></a><span class="do">## 996930 SNPs remain after merging OA and BMI summary statistics</span></span>
<span id="cb3-46"><a href="#cb3-46" tabindex="-1"></a><span class="do">## Results for genetic covariance between: OA and BMI</span></span>
<span id="cb3-47"><a href="#cb3-47" tabindex="-1"></a><span class="do">## Mean Z*Z: 0.6241</span></span>
<span id="cb3-48"><a href="#cb3-48" tabindex="-1"></a><span class="do">## Cross trait Intercept: 0.0446 (0.0112)</span></span>
<span id="cb3-49"><a href="#cb3-49" tabindex="-1"></a><span class="do">## Total Observed Scale Genetic Covariance (g_cov): 0.0394 (0.0019)</span></span>
<span id="cb3-50"><a href="#cb3-50" tabindex="-1"></a><span class="do">## g_cov Z: 20.5</span></span>
<span id="cb3-51"><a href="#cb3-51" tabindex="-1"></a><span class="do">## g_cov P-value: 8.6266e-94</span></span>
<span id="cb3-52"><a href="#cb3-52" tabindex="-1"></a><span class="do">##           </span></span>
<span id="cb3-53"><a href="#cb3-53" tabindex="-1"></a><span class="do">## Estimating heritability [5/10] for: T2D</span></span>
<span id="cb3-54"><a href="#cb3-54" tabindex="-1"></a><span class="do">## Heritability Results for trait: T2D</span></span>
<span id="cb3-55"><a href="#cb3-55" tabindex="-1"></a><span class="do">## Mean Chi^2 across remaining SNPs: 2.1342</span></span>
<span id="cb3-56"><a href="#cb3-56" tabindex="-1"></a><span class="do">## Lambda GC: 1.4961</span></span>
<span id="cb3-57"><a href="#cb3-57" tabindex="-1"></a><span class="do">## Intercept: 1.0054 (0.0259)</span></span>
<span id="cb3-58"><a href="#cb3-58" tabindex="-1"></a><span class="do">## Ratio: 0.0048 (0.0229)</span></span>
<span id="cb3-59"><a href="#cb3-59" tabindex="-1"></a><span class="do">## Total Observed Scale h2: 0.1307 (0.0055)</span></span>
<span id="cb3-60"><a href="#cb3-60" tabindex="-1"></a><span class="do">## h2 Z: 23.8</span></span>
<span id="cb3-61"><a href="#cb3-61" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-62"><a href="#cb3-62" tabindex="-1"></a><span class="do">## Calculating genetic covariance [6/10] for traits: T2D and BPH</span></span>
<span id="cb3-63"><a href="#cb3-63" tabindex="-1"></a><span class="do">## 1151699 SNPs remain after merging T2D and BPH summary statistics</span></span>
<span id="cb3-64"><a href="#cb3-64" tabindex="-1"></a><span class="do">## Results for genetic covariance between: T2D and BPH</span></span>
<span id="cb3-65"><a href="#cb3-65" tabindex="-1"></a><span class="do">## Mean Z*Z: 0.0113</span></span>
<span id="cb3-66"><a href="#cb3-66" tabindex="-1"></a><span class="do">## Cross trait Intercept: -0.0058 (0.0069)</span></span>
<span id="cb3-67"><a href="#cb3-67" tabindex="-1"></a><span class="do">## Total Observed Scale Genetic Covariance (g_cov): 0.0042 (0.0027)</span></span>
<span id="cb3-68"><a href="#cb3-68" tabindex="-1"></a><span class="do">## g_cov Z: 1.55</span></span>
<span id="cb3-69"><a href="#cb3-69" tabindex="-1"></a><span class="do">## g_cov P-value: 0.12162</span></span>
<span id="cb3-70"><a href="#cb3-70" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-71"><a href="#cb3-71" tabindex="-1"></a><span class="do">## Calculating genetic covariance [7/10] for traits: T2D and BMI</span></span>
<span id="cb3-72"><a href="#cb3-72" tabindex="-1"></a><span class="do">## 996408 SNPs remain after merging T2D and BMI summary statistics</span></span>
<span id="cb3-73"><a href="#cb3-73" tabindex="-1"></a><span class="do">## Results for genetic covariance between: T2D and BMI</span></span>
<span id="cb3-74"><a href="#cb3-74" tabindex="-1"></a><span class="do">## Mean Z*Z: 1.0387</span></span>
<span id="cb3-75"><a href="#cb3-75" tabindex="-1"></a><span class="do">## Cross trait Intercept: 0.0487 (0.0152)</span></span>
<span id="cb3-76"><a href="#cb3-76" tabindex="-1"></a><span class="do">## Total Observed Scale Genetic Covariance (g_cov): 0.0894 (0.0038)</span></span>
<span id="cb3-77"><a href="#cb3-77" tabindex="-1"></a><span class="do">## g_cov Z: 23.4</span></span>
<span id="cb3-78"><a href="#cb3-78" tabindex="-1"></a><span class="do">## g_cov P-value: 7.1286e-121</span></span>
<span id="cb3-79"><a href="#cb3-79" tabindex="-1"></a><span class="do">##           </span></span>
<span id="cb3-80"><a href="#cb3-80" tabindex="-1"></a><span class="do">## Estimating heritability [8/10] for: BPH</span></span>
<span id="cb3-81"><a href="#cb3-81" tabindex="-1"></a><span class="do">## Heritability Results for trait: BPH</span></span>
<span id="cb3-82"><a href="#cb3-82" tabindex="-1"></a><span class="do">## Mean Chi^2 across remaining SNPs: 1.1591</span></span>
<span id="cb3-83"><a href="#cb3-83" tabindex="-1"></a><span class="do">## Lambda GC: 0.9517</span></span>
<span id="cb3-84"><a href="#cb3-84" tabindex="-1"></a><span class="do">## Intercept: 0.9102 (0.0108)</span></span>
<span id="cb3-85"><a href="#cb3-85" tabindex="-1"></a><span class="do">## Ratio: -0.5646 (0.0676)</span></span>
<span id="cb3-86"><a href="#cb3-86" tabindex="-1"></a><span class="do">## Total Observed Scale h2: 0.0623 (0.0051)</span></span>
<span id="cb3-87"><a href="#cb3-87" tabindex="-1"></a><span class="do">## h2 Z: 12.2</span></span>
<span id="cb3-88"><a href="#cb3-88" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-89"><a href="#cb3-89" tabindex="-1"></a><span class="do">## Calculating genetic covariance [9/10] for traits: BPH and BMI</span></span>
<span id="cb3-90"><a href="#cb3-90" tabindex="-1"></a><span class="do">## 1001610 SNPs remain after merging BPH and BMI summary statistics</span></span>
<span id="cb3-91"><a href="#cb3-91" tabindex="-1"></a><span class="do">## Results for genetic covariance between: BPH and BMI</span></span>
<span id="cb3-92"><a href="#cb3-92" tabindex="-1"></a><span class="do">## Mean Z*Z: 0.0049</span></span>
<span id="cb3-93"><a href="#cb3-93" tabindex="-1"></a><span class="do">## Cross trait Intercept: 0.01 (0.0101)</span></span>
<span id="cb3-94"><a href="#cb3-94" tabindex="-1"></a><span class="do">## Total Observed Scale Genetic Covariance (g_cov): -6e-04 (0.0033)</span></span>
<span id="cb3-95"><a href="#cb3-95" tabindex="-1"></a><span class="do">## g_cov Z: -0.175</span></span>
<span id="cb3-96"><a href="#cb3-96" tabindex="-1"></a><span class="do">## g_cov P-value: 0.86117</span></span>
<span id="cb3-97"><a href="#cb3-97" tabindex="-1"></a><span class="do">##           </span></span>
<span id="cb3-98"><a href="#cb3-98" tabindex="-1"></a><span class="do">## Estimating heritability [10/10] for: BMI</span></span>
<span id="cb3-99"><a href="#cb3-99" tabindex="-1"></a><span class="do">## Heritability Results for trait: BMI</span></span>
<span id="cb3-100"><a href="#cb3-100" tabindex="-1"></a><span class="do">## Mean Chi^2 across remaining SNPs: 3.9344</span></span>
<span id="cb3-101"><a href="#cb3-101" tabindex="-1"></a><span class="do">## Lambda GC: 2.7869</span></span>
<span id="cb3-102"><a href="#cb3-102" tabindex="-1"></a><span class="do">## Intercept: 1.0199 (0.0277)</span></span>
<span id="cb3-103"><a href="#cb3-103" tabindex="-1"></a><span class="do">## Ratio: 0.0068 (0.0094)</span></span>
<span id="cb3-104"><a href="#cb3-104" tabindex="-1"></a><span class="do">## Total Observed Scale h2: 0.2091 (0.0063)</span></span>
<span id="cb3-105"><a href="#cb3-105" tabindex="-1"></a><span class="do">## h2 Z: 33.3</span></span>
<span id="cb3-106"><a href="#cb3-106" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-107"><a href="#cb3-107" tabindex="-1"></a><span class="do">##       </span></span>
<span id="cb3-108"><a href="#cb3-108" tabindex="-1"></a><span class="do">## Genetic Correlation Results</span></span>
<span id="cb3-109"><a href="#cb3-109" tabindex="-1"></a><span class="do">## Genetic Correlation between OA and T2D: 0.2216 (0.0208)</span></span>
<span id="cb3-110"><a href="#cb3-110" tabindex="-1"></a><span class="do">## Genetic Correlation between OA and BPH: 0.1295 (0.0333)</span></span>
<span id="cb3-111"><a href="#cb3-111" tabindex="-1"></a><span class="do">## Genetic Correlation between OA and BMI: 0.4361 (0.0182)</span></span>
<span id="cb3-112"><a href="#cb3-112" tabindex="-1"></a><span class="do">## Genetic Correlation between T2D and BPH: 0.0462 (0.0303)</span></span>
<span id="cb3-113"><a href="#cb3-113" tabindex="-1"></a><span class="do">## Genetic Correlation between T2D and BMI: 0.5408 (0.022)</span></span>
<span id="cb3-114"><a href="#cb3-114" tabindex="-1"></a><span class="do">## Genetic Correlation between BPH and BMI: -0.005 (0.0288)</span></span>
<span id="cb3-115"><a href="#cb3-115" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-116"><a href="#cb3-116" tabindex="-1"></a><span class="do">##       </span></span>
<span id="cb3-117"><a href="#cb3-117" tabindex="-1"></a><span class="do">## Partial genetic Correlation Results</span></span>
<span id="cb3-118"><a href="#cb3-118" tabindex="-1"></a><span class="do">## Partial genetic Correlation between OA and T2D: -0.0189 (0.0296)</span></span>
<span id="cb3-119"><a href="#cb3-119" tabindex="-1"></a><span class="do">## Partial genetic Correlation between OA and BPH: 0.1463 (0.0344)</span></span>
<span id="cb3-120"><a href="#cb3-120" tabindex="-1"></a><span class="do">## Partial genetic Correlation between T2D and BPH: 0.0581 (0.0315)</span></span>
<span id="cb3-121"><a href="#cb3-121" tabindex="-1"></a><span class="do">##      </span></span>
<span id="cb3-122"><a href="#cb3-122" tabindex="-1"></a><span class="do">## Analysis finished running at 15:14</span></span>
<span id="cb3-123"><a href="#cb3-123" tabindex="-1"></a><span class="do">## Runtime: 0 minute(s) and 55 second(s)</span></span>
<span id="cb3-124"><a href="#cb3-124" tabindex="-1"></a><span class="do">## </span></span></code></pre></div>
</details><!--
- **Example B**
add example with multiple confounder one implemented
-->
</div>
<div class="section level3">
<h3 id="results">Results<a class="anchor" aria-label="anchor" href="#results"></a>
</h3>
<p><strong><code>ldsc_partial()</code></strong> returns a named list containing the following results:</p>
<ul>
<li>
<code>res_diff</code> (pairwise results)</li>
</ul>
<p><code>condition.1</code> : first condition in the pair,<br><code>condition.2</code> : second condition in the pair,<br><code>rg</code> : unadjusted genetic correlation between the two conditions,<br><code>rg.SE</code> : standard error of the unadjusted genetic correlation between the two conditions,<br><code>partial_rg</code> : partial genetic correlation between the two conditions,<br><code>partial_rg.SE</code> : standard error of the partial genetic correlation between the two conditions,<br><code>rg_cov</code> : covariance between the unadjusted and the partial correlation estimates for the pair,<br><code>diff.T</code> : test statistic used to test for the difference between the unadjusted and the partial correlation estimates for the pair,<br><code>diff.P</code> : p-value corresponding to the test statistic used to test the difference between the unadjusted and the partial correlation estimates for the pair.</p>
<ul>
<li><p><code>S</code> : estimated genetic covariance matrix for all conditions + confounder.</p></li>
<li><p><code>V</code> : variance covariance matrix of the parameter estimates in <code>S</code>.</p></li>
<li><p><code>S_Stand</code> : estimated genetic correlation matrix for all conditions + confounder.</p></li>
<li><p><code>V_Stand</code> : variance covariance matrix of the parameter estimates in <code>S_Stand</code>.</p></li>
<li><p><code>partial.S</code> : estimated partial genetic covariance matrix for all conditions.</p></li>
<li><p><code>partial.V</code> : variance covariance matrix of the parameter estimates in <code>partial.S</code>.</p></li>
<li><p><code>partial.S_Stand</code> : estimated partial genetic correlation matrix for all conditions.</p></li>
<li><p><code>partial.V_Stand</code> : variance covariance matrix of the parameter estimates in <code>partial.S_Stand</code>.</p></li>
<li><p><code>I</code> : matrix containing the cross-trait intercepts.</p></li>
</ul>
<div class="section level5">
<h5 id="aditionnaly-log-and-results-files-are-created-in-the-current-working-directory-">Aditionnaly, log and results files are created in the current working directory :<a class="anchor" aria-label="anchor" href="#aditionnaly-log-and-results-files-are-created-in-the-current-working-directory-"></a>
</h5>
<ul>
<li><p>**&lt;log.name&gt;_ldsc.log** - log file</p></li>
<li><p>**&lt;log.name&gt;_difference.tsv** - results file</p></li>
<li><p><strong>Example A</strong></p></li>
</ul>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">### structure of the results</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/str.html" class="external-link">str</a></span><span class="op">(</span><span class="va">A</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" tabindex="-1"></a><span class="do">## List of 10</span></span>
<span id="cb5-2"><a href="#cb5-2" tabindex="-1"></a><span class="do">##  $ res_diff       :'data.frame': 3 obs. of  9 variables:</span></span>
<span id="cb5-3"><a href="#cb5-3" tabindex="-1"></a><span class="do">##   ..$ condition.1  : chr [1:3] "OA" "OA" "T2D"</span></span>
<span id="cb5-4"><a href="#cb5-4" tabindex="-1"></a><span class="do">##   ..$ condition.2  : chr [1:3] "T2D" "BPH" "BPH"</span></span>
<span id="cb5-5"><a href="#cb5-5" tabindex="-1"></a><span class="do">##   ..$ rg           : num [1:3] 0.2216 0.1295 0.0462</span></span>
<span id="cb5-6"><a href="#cb5-6" tabindex="-1"></a><span class="do">##   ..$ rg.SE        : num [1:3] 0.0208 0.0333 0.0303</span></span>
<span id="cb5-7"><a href="#cb5-7" tabindex="-1"></a><span class="do">##   ..$ partial_rg   : num [1:3] -0.0189 0.1463 0.0581</span></span>
<span id="cb5-8"><a href="#cb5-8" tabindex="-1"></a><span class="do">##   ..$ partial_rg.SE: num [1:3] 0.0296 0.0344 0.0315</span></span>
<span id="cb5-9"><a href="#cb5-9" tabindex="-1"></a><span class="do">##   ..$ rg_cov       : num [1:3] 0.000451 0.001059 0.000819</span></span>
<span id="cb5-10"><a href="#cb5-10" tabindex="-1"></a><span class="do">##   ..$ diff.T       : num [1:3] 11.92 -1.29 -0.72</span></span>
<span id="cb5-11"><a href="#cb5-11" tabindex="-1"></a><span class="do">##   ..$ diff.P       : num [1:3] 9.03e-33 1.96e-01 4.71e-01</span></span>
<span id="cb5-12"><a href="#cb5-12" tabindex="-1"></a><span class="do">##  $ S              : num [1:4, 1:4] 0.03902 0.01582 0.00639 0.0394 0.01582 ...</span></span>
<span id="cb5-13"><a href="#cb5-13" tabindex="-1"></a><span class="do">##   ..- attr(*, "dimnames")=List of 2</span></span>
<span id="cb5-14"><a href="#cb5-14" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:4] "OA" "T2D" "BPH" "BMI"</span></span>
<span id="cb5-15"><a href="#cb5-15" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:4] "OA" "T2D" "BPH" "BMI"</span></span>
<span id="cb5-16"><a href="#cb5-16" tabindex="-1"></a><span class="do">##  $ V              : num [1:10, 1:10] 1.94e-06 6.56e-07 -5.20e-09 8.45e-07 2.01e-06 ...</span></span>
<span id="cb5-17"><a href="#cb5-17" tabindex="-1"></a><span class="do">##  $ S_Stand        : num [1:4, 1:4] 1 0.222 0.129 0.436 0.222 ...</span></span>
<span id="cb5-18"><a href="#cb5-18" tabindex="-1"></a><span class="do">##   ..- attr(*, "dimnames")=List of 2</span></span>
<span id="cb5-19"><a href="#cb5-19" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:4] "OA" "T2D" "BPH" "BMI"</span></span>
<span id="cb5-20"><a href="#cb5-20" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:4] "OA" "T2D" "BPH" "BMI"</span></span>
<span id="cb5-21"><a href="#cb5-21" tabindex="-1"></a><span class="do">##  $ V_Stand        : num [1:10, 1:10] 1.93e-30 2.08e-18 -5.98e-19 1.07e-18 7.29e-33 ...</span></span>
<span id="cb5-22"><a href="#cb5-22" tabindex="-1"></a><span class="do">##  $ partial.S      : num [1:3, 1:3] 0.0316 -0.00102 0.00649 -0.00102 0.09248 ...</span></span>
<span id="cb5-23"><a href="#cb5-23" tabindex="-1"></a><span class="do">##   ..- attr(*, "dimnames")=List of 2</span></span>
<span id="cb5-24"><a href="#cb5-24" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:3] "OA" "T2D" "BPH"</span></span>
<span id="cb5-25"><a href="#cb5-25" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:3] "OA" "T2D" "BPH"</span></span>
<span id="cb5-26"><a href="#cb5-26" tabindex="-1"></a><span class="do">##  $ partial.V      : num [1:6, 1:6] 1.83e-06 5.95e-07 3.32e-07 1.97e-06 2.00e-07 ...</span></span>
<span id="cb5-27"><a href="#cb5-27" tabindex="-1"></a><span class="do">##  $ partial.S_Stand: num [1:3, 1:3] 1 -0.0189 0.1463 -0.0189 1 ...</span></span>
<span id="cb5-28"><a href="#cb5-28" tabindex="-1"></a><span class="do">##   ..- attr(*, "dimnames")=List of 2</span></span>
<span id="cb5-29"><a href="#cb5-29" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:3] "OA" "T2D" "BPH"</span></span>
<span id="cb5-30"><a href="#cb5-30" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:3] "OA" "T2D" "BPH"</span></span>
<span id="cb5-31"><a href="#cb5-31" tabindex="-1"></a><span class="do">##  $ partial.V_Stand: num [1:6, 1:6] 0 0 0 0 0 ...</span></span>
<span id="cb5-32"><a href="#cb5-32" tabindex="-1"></a><span class="do">##  $ I              : num [1:4, 1:4] 0.8991 0.0299 0.0418 0.0446 0.0299 ...</span></span>
<span id="cb5-33"><a href="#cb5-33" tabindex="-1"></a><span class="do">##   ..- attr(*, "dimnames")=List of 2</span></span>
<span id="cb5-34"><a href="#cb5-34" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:4] "OA" "T2D" "BPH" "BMI"</span></span>
<span id="cb5-35"><a href="#cb5-35" tabindex="-1"></a><span class="do">##   .. ..$ : chr [1:4] "OA" "T2D" "BPH" "BMI"</span></span></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">### matrix of unadjusted genetic correlations</span></span>
<span></span>
<span><span class="va">A</span><span class="op">$</span><span class="va">S_Stand</span></span></code></pre></div>
<div class="sourceCode" id="cb7"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" tabindex="-1"></a><span class="do">##            OA        T2D          BPH          BMI</span></span>
<span id="cb7-2"><a href="#cb7-2" tabindex="-1"></a><span class="do">## OA  1.0000000 0.22155702  0.129499318  0.436123972</span></span>
<span id="cb7-3"><a href="#cb7-3" tabindex="-1"></a><span class="do">## T2D 0.2215570 1.00000000  0.046169361  0.540761961</span></span>
<span id="cb7-4"><a href="#cb7-4" tabindex="-1"></a><span class="do">## BPH 0.1294993 0.04616936  1.000000000 -0.005027793</span></span>
<span id="cb7-5"><a href="#cb7-5" tabindex="-1"></a><span class="do">## BMI 0.4361240 0.54076196 -0.005027793  1.000000000</span></span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">### pairwise results</span></span>
<span><span class="co"># in this case, we observed a significant difference between the unadjusted and the partial </span></span>
<span><span class="co"># genetic correlation estimates only for OA and T2D. </span></span>
<span><span class="co"># This does make sense since the genetic correlation between BMI and BPH is very low, </span></span>
<span><span class="co"># explaining why adjusting for BMI does not affect the genetic correlation between BPH </span></span>
<span><span class="co"># and the other conditions.</span></span>
<span></span>
<span><span class="va">A</span><span class="op">$</span><span class="va">res_diff</span></span></code></pre></div>
<div class="sourceCode" id="cb9"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" tabindex="-1"></a><span class="do">##   condition.1 condition.2         rg      rg.SE  partial_rg partial_rg.SE</span></span>
<span id="cb9-2"><a href="#cb9-2" tabindex="-1"></a><span class="do">## 1          OA         T2D 0.22155702 0.02076908 -0.01886781    0.02961074</span></span>
<span id="cb9-3"><a href="#cb9-3" tabindex="-1"></a><span class="do">## 2          OA         BPH 0.12949932 0.03327598  0.14634480    0.03435472</span></span>
<span id="cb9-4"><a href="#cb9-4" tabindex="-1"></a><span class="do">## 3         T2D         BPH 0.04616936 0.03030614  0.05811963    0.03154903</span></span>
<span id="cb9-5"><a href="#cb9-5" tabindex="-1"></a><span class="do">##         rg_cov     diff.T       diff.P</span></span>
<span id="cb9-6"><a href="#cb9-6" tabindex="-1"></a><span class="do">## 1 0.0004507507 11.9225545 9.029680e-33</span></span>
<span id="cb9-7"><a href="#cb9-7" tabindex="-1"></a><span class="do">## 2 0.0010588452 -1.2925710 1.961595e-01</span></span>
<span id="cb9-8"><a href="#cb9-8" tabindex="-1"></a><span class="do">## 3 0.0008192339 -0.7201883 4.714090e-01</span></span></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">### functions to list conditions / pairs</span></span>
<span></span>
<span><span class="fu"><a href="reference/get_conditions.html">get_conditions</a></span><span class="op">(</span><span class="va">A</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" tabindex="-1"></a><span class="do">## [1] "OA"  "T2D" "BPH"</span></span></code></pre></div>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="reference/get_pairs.html">get_pairs</a></span><span class="op">(</span><span class="va">A</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb13"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" tabindex="-1"></a><span class="do">## [1] "OA-T2D"  "OA-BPH"  "T2D-BPH"</span></span></code></pre></div>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">### forest plot</span></span>
<span><span class="fu"><a href="reference/forest_plot.html">forest_plot</a></span><span class="op">(</span><span class="va">A</span><span class="op">)</span></span></code></pre></div>
<p><img src="reference/figures/README-resultsA-1.png" width="100%"></p>
</div>
</div>
</div>
<div class="section level2">
<h2 id="runtime">Runtime<a class="anchor" aria-label="anchor" href="#runtime"></a>
</h2>
<p>Example A ~ 0 minute(s) and 55 second(s)</p>
<p>Runtime can be influenced by the number of traits.</p>
<p><font color="grey"><small> Results from analyses performed on a Windows 10 laptop - Processor : Intel(R) Core(TM) i5-10310U CPU @ 1.70GHz 2.21 GHz - Memory : 16.0 GB. </small></font></p>
<!-- ## Contributors -->
</div>
<div class="section level2">
<h2 id="citation">Citation<a class="anchor" aria-label="anchor" href="#citation"></a>
</h2>
<p>If you use the <code>partialLDSC</code> package, please cite:</p>
<blockquote>
<p>Mounier <em>et al.</em> (2024) Genetics identifies obesity as a shared risk factor for co-occurring multiple long-term conditions. medRxiv <a href="https://doi.org/10.1101/2024.07.10.24309772" class="external-link">https://doi.org/10.1101/2024.07.10.24309772</a></p>
</blockquote>
</div>
<div class="section level2">
<h2 id="contact">Contact<a class="anchor" aria-label="anchor" href="#contact"></a>
</h2>
<p>… <!-- <mounier.ninon@gmail.com> --></p>
</div>
</div>
  </main><aside class="col-md-3"><div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li><a href="LICENSE.html">Full license</a></li>
<li><small>GPL (&gt;= 3)</small></li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing partialLDSC</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>
<a href="https://github.com/n-mounier" class="external-link">Ninon Mounier</a> <br><small class="roles"> Author, maintainer </small>  </li>
<li>
<a href="https://lcpilling.github.io" class="external-link">Luke Pilling</a> <br><small class="roles"> Author </small> <a href="https://orcid.org/0000-0002-3332-8454" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
<li>Bethany Voller <br><small class="roles"> Author </small>  </li>
</ul>
</div>

<div class="dev-status">
<h2 data-toc-skip>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://github.com/GEMINI-multimorbidity/partialLDSC" class="external-link"><img src="https://img.shields.io/badge/version-0.1.1-informational.svg"></a></li>
<li><a href="https://github.com/GEMINI-multimorbidity/partialLDSC/commits/master" class="external-link"><img src="https://img.shields.io/github/last-commit/GEMINI-multimorbidity/partialLDSC.svg"></a></li>
<li><a href="https://www.tidyverse.org/lifecycle/#experimental" class="external-link"><img src="https://img.shields.io/badge/lifecycle-experimental-orange"></a></li>
<li><a href="https://zenodo.org/doi/10.5281/zenodo.12721532" class="external-link"><img src="https://zenodo.org/badge/716022171.svg" alt="DOI"></a></li>
</ul>
</div>

  </aside>
</div>


    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by <a href="https://github.com/n-mounier" class="external-link">Ninon Mounier</a>, <a href="https://lcpilling.github.io" class="external-link">Luke Pilling</a>, Bethany Voller.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
